### PR TITLE
Fixes #88 - Message misses "edited" field

### LINKED ISSUE #48: delete user method
I cannot find this call in the API: https://api.slack.com/scim#delete-users-id is it not implemented?

### LINKED ISSUE #88: Message misses "edited" field
Message misses "edited" field

Can you add it?

### PR DESCRIPTION
**Pull request by Li0n13**
Fixes #88 - Message misses "edited" field

• add "edited" field to Message
• add test

### CODE REVIEW

**Comment by seratch** [comment]
Thanks, could you fix the commit message as well? (not 48 but 88)

File: src/main/java/com/github/seratch/jslack/api/model/Message.java [hunk 1]
```diff
@@ -39,6 +41,12 @@
     /**
      * The root message information of a "thread_broadcast" message.
      */
+    @Data
```
Comment (seratch) [inline src/main/java/com/github/seratch/jslack/api/model/Message.java]: >The root message information of a "thread_broadcast" message.

This class comment for MessageRoot. Please don't put Edited here.

**Comment by Li0n13** [comment]
Moved Edited class. Fixed commit message.

**Comment by seratch** [comment]
Thanks!

**Comment by Li0n13** [comment]
Thank you!

**Comment by seratch** [comment]
Just released in version 1.1.7

**Pull request merged**